tests/integration/docker/test.common

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+DOCKER_EXE="docker"
+DOCKER_SERVICE="docker-cor"
```

Comment:
I reviewed the documentation for other distributions and clearlinux is the only one that is using docker-cor i.e. #systemctl restart docker-cor so one way to solve this is to try to figure out if you are using clearlinux or other Linux distribution. Make a distinction in order to do #systemctl restart docker-cor (for clear) or #systemctl restart docker (other distributions). Also, I verified that when you are running a container in other Linux distribution and you performed a #docker info,  it appears as Default runtime: cor the same as the clearlinux case so we do not have issues in that.